Exebite.DataAccess/Repositories/CommentRepository/Model/CommentUpdateModel.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Exebite.DataAccess.Repositories.CommentRepository.Model
+{
+    public class CommentUpdateModel
+    {
+        public long MealId { get; set; }
```

Comment:
It is considered a good practice to add the primary key of the entity that is going to be updated to the request model, too. This value would also need to be populated by the FE team.

```suggestion
        public long MealId { get; set; }
        
        public int CommentId { get; set; }
```